src/Equinox/Flow.fs

```diff
@@ -64,9 +66,12 @@ module internal Flow =
         (syncState : SyncState<'event, 'state>)
         (decide : 'state -> Async<'result * 'event list>)
         : Async<'result> =
-        if maxSyncAttempts < 1 then raise <| System.ArgumentOutOfRangeException("maxSyncAttempts", maxSyncAttempts, "should be >= 1")
+
+        if maxSyncAttempts < 1 then 
```

Comment:
again, guard clauses as one lines are kinda intentional - do you not agree they are idiomatic ? (if it confused you, then you're the judge of course)

Reply:
It depends to be honest, I am not set in stone about them, whatever improves the readability wins in my case, it all depends on how long the `then` part is

Reply:
I am doing a soft max len 120 and hard at 160 normally, so that's well inside limits (I can feel Seemann and Henney's frowns though)